dashboard/config/routes.rb

```diff
@@ -23,6 +23,10 @@ def redirect_to_teacher_dashboard
     end
   end
 
+
+  # Media proxying
+  get 'media', to: 'media_proxy#get', format: false
```

Comment:
Why is `format: false` needed here? I'm looking at adding a similar proxy for `startWebRequest`